packages/validator/src/cli-validator/utils/print-results.js

```diff
@@ -88,36 +81,38 @@ module.exports = function print(
         const lineNumber = getLineNumberForPath(originalFile, path);
 
         // print the path array as a dot-separated string
-
-        logger.info(chalk[color](`  Message :   ${problem.message}`));
-        if (verbose && problem.rule) {
-          logger.info(chalk[color](`  Rule    :   ${problem.rule}`));
+        if (!summaryOnly) {
+          logger.info(chalk[color](`  Message :   ${problem.message}`));
+          if (problem.rule) {
+            logger.info(chalk[color](`  Rule    :   ${problem.rule}`));
+          }
+          logger.info(chalk[color](`  Path    :   ${path.join('.')}`));
+          logger.info(chalk[color](`  Line    :   ${lineNumber}`));
+          if (verbose && problem.componentPath) {
+            const componentPath = getPathAsArray(problem.componentPath);
+            const componentLine = getLineNumberForPath(
+              originalFile,
+              componentPath
+            );
+            logger.info(
+              chalk[color](`  Component Path    :   ${componentPath.join('.')}`)
+            );
+            logger.info(
+              chalk[color](`  Component Line    :   ${componentLine}`)
+            );
+          }
+          logger.info('');
         }
-        logger.info(chalk[color](`  Path    :   ${path.join('.')}`));
-        logger.info(chalk[color](`  Line    :   ${lineNumber}`));
-        if (verbose && problem.componentPath) {
-          const componentPath = getPathAsArray(problem.componentPath);
-          const componentLine = getLineNumberForPath(
-            originalFile,
-            componentPath
-          );
-          logger.info(
-            chalk[color](`  Component Path    :   ${componentPath.join('.')}`)
-          );
-          logger.info(chalk[color](`  Component Line    :   ${componentLine}`));
-        }
-        logger.info('');
       });
     });
   });
 
-  // print the stats here, if applicable
+  // Print the summary here if we had any messages.
   if (
-    reportingStats &&
-    (stats.errors.total ||
-      stats.warnings.total ||
-      stats.infos.total ||
-      stats.hints.total)
+    stats.errors.total ||
+    stats.warnings.total ||
+    stats.infos.total ||
+    stats.hints.total
   ) {
     logger.info(chalk.bgCyan('statistics\n'));
```

Comment:
Can we change this to be "summary" since that's the language we're using now?

```suggestion
    logger.info(chalk.bgCyan('summary\n'));
```

Reply:
latest commit fixes this